pkg/client/cloudevents/source_client.go

```diff
@@ -144,12 +115,16 @@ func (s *SourceClientImpl) OnDelete(ctx context.Context, id string) error {
 	return nil
 }
 
-func (s *SourceClientImpl) Resync(ctx context.Context) error {
+func (s *SourceClientImpl) Subscribe(ctx context.Context, handlers ...cegeneric.ResourceHandler[*api.Resource]) {
+	s.CloudEventSourceClient.Subscribe(ctx, handlers...)
+}
+
+func (s *SourceClientImpl) Resync(ctx context.Context, sourceID, consumerID string) error {
```

Comment:
with new sdk-go, the sourceID will not be needed

Reply:
updated the sdk-go dependency, removed sourceID.